### PR TITLE
Add SPI change to 453 release notes

### PR DESCRIPTION
**Pull request by mosabua**
## Description

One follow up for https://trinodb.slack.com/archives/C07ABNN828M/p1721943087420349 

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

File: docs/src/main/sphinx/release/release-453.md
Comment (martint) [inline docs/src/main/sphinx/release/release-453.md:144]: This is not (yet) a breaking change. The old method is still available.

Comment (mosabua) [reply docs/src/main/sphinx/release/release-453.md:144]: Oh .. so it was only soft deprecated?

Comment (mosabua) [reply docs/src/main/sphinx/release/release-453.md:144]: @mosiac1 ?

Comment (mosabua) [reply docs/src/main/sphinx/release/release-453.md]: confirmed on slack

**Review by martint — APPROVED** [review]

File: docs/src/main/sphinx/release/release-453.md
Comment (ebyhr) [inline docs/src/main/sphinx/release/release-453.md:144]: I don't think #21997 added `getTableColumnMasks` method. https://github.com/trinodb/trino/pull/21997/files#diff-9c1c468145e70554a262a95c50bd9bdd474935e69663e803df567939f0b4da0cR967

**Review by ebyhr — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/release/release-453.md
Comment (martint) [reply docs/src/main/sphinx/release/release-453.md:144]: Ah, yes, the PR originally had it as getTableColumnMasks but we renamed it before merging. Good catch!

Comment (mosabua) [reply docs/src/main/sphinx/release/release-453.md]: Nice! Adjusted to that now and made the commit message longer ;-)

**Comment by mosabua** [comment]
Will merge after approval from @ebyhr

**Review by ebyhr — APPROVED** [review]

**Pull request merged**
